compiler/testData/diagnostics/tests/secondaryConstructors/useOfPropertiesWithoutPrimary.fir.kt

```diff
@@ -9,7 +9,11 @@ class A {
 
     val useInitialized = useUnitialized + v
 
-    val uninitialized: Int
+    <!MUST_BE_INITIALIZED_OR_BE_ABSTRACT!>val uninitialized: Int<!>
+
```

Comment:
Why did you move this constructor?

Reply:
To showcase constructors that are not topologically sorted, as mentioned at https://github.com/JetBrains/kotlin/pull/4142#discussion_r583108273. Indeed, this change works on FE 1.0, but didn't for my initial change. Now works well with topological sorting.